repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -2038,3 +2139,40 @@ export const getProductGroupByNocAndId = async (noc: string, id: number): Promis
         return undefined;
     }
 };
+
+export const getProductGroupByNameAndNocCode = async (
```

Comment:
Why are we getting a product group by using the name and not the ID?

Reply:
to have a unique name per user

Reply:
we always don't get the id on the page as it is the same page for creation also.

Reply:
But if they're creating a group, they wont be calling getProductGroupByNameAndNocCode will they?